build_tools/i18n/utils.py

```diff
@@ -51,16 +52,24 @@
     KEY_DEFAULT_FONT: "NotoSans",
 }
 
+# Keys used in simple internal 'font info' dictionaries
+KEY_NAME = "name"
+KEY_HAS_UI = "has_ui_variant"
+
 
 def to_locale(language):
     """
     Turns a language name (en-us) into a locale name (en_US).
+    Logic is derived from Django so be careful about changing it.
```

Comment:
I enjoy the juxtaposition of this comment and the extensive diff that follows it :)

Reply:
ha that was the `black` autoformatter